The job **`i686-gnu-1`** failed! Check out the build log: [(web)](https://github.com/rust-lang/rust/actions/runs/18073213860/job/51425982473) [(plain enhanced)](https://triage.rust-lang.org/gha-logs/rust-lang/rust/51425982473) [(plain)](https://github.com/rust-lang/rust/commit/c5ca5297e5438a90da44b4285551e316a93d9ede/checks/51425982473/logs)

<details><summary><i>Click to see the possible cause of the failure (guessed by this bot)</i></summary>

```plain
test [crashes] tests/crashes/140729.rs ... ok
test [crashes] tests/crashes/140860.rs ... ok
test [crashes] tests/crashes/140891.rs ... ok
test [crashes] tests/crashes/144293-indirect-ops-llvm.rs ... ok
2025-09-28T12:14:09.669189Z ERROR compiletest::runtest: fatal error, panic: "crashtest no longer crashes/triggers ICE, hooray! Please give it a meaningful name, add a doc-comment to the start of the test explaining why it exists and move it to tests/ui or wherever you see fit. Adding 'Fixes #<issueNr>' to your PR description ensures that the corresponding ticket is auto-closed upon merge. If you want to see verbose output, set `COMPILETEST_VERBOSE_CRASHES=1`."
test [crashes] tests/crashes/34127.rs ... FAILED
test [crashes] tests/crashes/87577.rs ... ok
test [crashes] tests/crashes/54888.rs ... ok
test [crashes] tests/crashes/141124.rs ... ok
test [crashes] tests/crashes/88296.rs ... ok
---

---- [crashes] tests/crashes/34127.rs stdout ----
------rustc stdout------------------------------

------rustc stderr------------------------------
error[E0080]: attempt to shift left by `63_i32`, which would overflow
##[error] --> /checkout/tests/crashes/34127.rs:5:15
  |
5 | let _a = [(); 1 << 63];
  |               ^^^^^^^ evaluation of `main::{constant#0}` failed here

error: aborting due to 1 previous error

For more information about this error, try `rustc --explain E0080`.

------------------------------------------

error: crashtest no longer crashes/triggers ICE, hooray! Please give it a meaningful name, add a doc-comment to the start of the test explaining why it exists and move it to tests/ui or wherever you see fit. Adding 'Fixes #<issueNr>' to your PR description ensures that the corresponding ticket is auto-closed upon merge. If you want to see verbose output, set `COMPILETEST_VERBOSE_CRASHES=1`.

thread '[crashes] tests/crashes/34127.rs' panicked at src/tools/compiletest/src/runtest/crashes.rs:17:18:
fatal error
stack backtrace:
   5: __rustc::rust_begin_unwind
```

</details>

--- a/tests/crashes/125772.rs
+++ b/tests/crashes/125772.rs
@@ -1,5 +1,5 @@
 //@ known-bug: rust-lang/rust#125772
-//@ only-x86_64
+//@ only-64bit
 #![feature(generic_const_exprs)]
 
 struct Outer<const A: i64, const B: i64>();

--- a/tests/crashes/131292.rs
+++ b/tests/crashes/131292.rs
@@ -1,5 +1,5 @@
 //@ known-bug: #131292
-//@ only-x86_64
+//@ needs-asm-support
 use std::arch::asm;
 
 unsafe fn f6() {

--- a/tests/crashes/34127.rs
+++ b/tests/crashes/34127.rs
@@ -1,6 +1,6 @@
 //@ compile-flags: -g -Copt-level=0 -Z verify-llvm-ir
 //@ known-bug: #34127
-//@ only-x86_64
+//@ only-64bit
 
 pub fn main() {
 let _a = [(); 1 << 63];

